tests/integration_tests/test_applications_from_config.py

```diff
@@ -28,6 +28,8 @@ def get_application_command(app, config_file=None):
         Name of the application.
     config_file: str
         Configuration file.
+    config_string: str
+        Configuration string (e.g., '--version')
```

Comment:
Any one string operations defined earlier. For now, this is version and help, but this might change. So I consciously chose a single example to not not 'hardwire' things into docstring. Isn't that enough to understand?

Reply:
I don't only comment when it is not understandable to me, but I comment to improve the clarity, especially for new developers who might join the team. It would be good to have this information you just mentioned in the docstring.